### PR TITLE
First Pull Request

### PR DESCRIPTION
**Pull request by mwanawabangona**
# In this feature update:

 - Use Gitflow
- Create an npm project with webpack.
- Write plain HTML markup with minimum styling using project wireframe as a guide.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+          </ul>
+        </div>
```
Comment (mikethreels) [inline src/index.html]: - [ ] good job creating the project and making it look like the wireframe 👍 however right now the code you wrote is completely in HTML the goal of the assignment is to make use of ES6 modules to write modular JavaScript with proper ES6 syntax. please create the list items using ES6 arrow functions. Please refer to [this article](https://www.w3schools.com/js/js_arrow_function.asp) for more information regarding arrow functions.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Ellis, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (henatan99) [reply src/index.html]: This issue has been addressed well! Good job! :clap: :clap: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @mwanawabangona , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (mwanawabangona) [reply src/index.html]: @mikethreels thank you for your review

**Pull request merged**